data/darktableconfig.xml.in

```diff
@@ -721,7 +721,7 @@
   <dtconfig prefs="gui" section="lighttable">
     <name>plugins/lighttable/extended_pattern</name>
     <type>string</type>
-    <default>$(FILE_NAME).$(FILE_EXTENSION)$(NL)$(EXIF_EXPOSURE) f/$(EXIF_APERTURE) $(EXIF_FOCAL_LENGTH)mm ISO $(EXIF_ISO)</default>
+    <default>$(FILE_NAME).$(FILE_EXTENSION)$(NL)$(EXIF_EXPOSURE) f/$(EXIF_APERTURE) $(EXIF_FOCAL_LENGTH)mm ISO $(EXIF_ISO)$(SIDECAR_TXT)</default>
```

Comment:
Using this to configure the txt is better indeed, but in that case I suppose we want to remove the old prefs which was used to activate or not the txt.

Reply:
no, this is still used in culling preview...